### PR TITLE
New package: python-exiv2-0.3.2

### PR DESCRIPTION
**Pull request by diogoleal**

### CODE REVIEW

File: srcpkgs/variety/template [hunk 1]
```diff
@@ -9,7 +9,7 @@ pycompile_module='variety'
 hostmakedepends="python-distutils-extra intltool"
 depends="desktop-file-utils gtk+3 ImageMagick libnotify python-BeautifulSoup4
  python-cairo python-configobj python-dbus exiv2 python-gobject python-httplib2
- python-Pillow python-lxml python-curl webkitgtk"
+ python-Pillow python-lxml python-curl python-exiv2 webkitgtk "
```
Comment (SirDoctors) [inline srcpkgs/variety/template]: Could you make this a separate commit? You should have a commit for each package you modify.

**Comment by asergi** [comment]
Obsolete. Not needed anymore since variety>=0.6.4. See https://bugs.launchpad.net/variety/+bug/1280349.
